pkgs/development/dotnet-modules/python-language-server/updater.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p gnused jq common-updater-scripts nuget-to-nix dotnet-sdk_3 nix-prefetch-git
+set -eo pipefail
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+deps_file="$(realpath ./deps.nix)"
+
+nix-prefetch-git https://github.com/microsoft/python-language-server --quiet > repo_info
+new_version="$(jq -r ".date" < repo_info | cut -d"T" -f1)"
+new_hash="$(jq -r ".sha256" < repo_info)"
+new_rev="$(jq -r ".rev" < repo_info)"
+rm repo_info
+
+old_rev="$(sed -nE 's/\s*rev = "(.*)".*/\1/p' ./default.nix)"
+
+if [[ "$new_rev" == "$old_rev" ]]; then
```

Comment:
```suggestion
if [[ $new_rev == $old_rev ]]; then
```
shellcheck would have a problem with this: quoting both sides has strange side effects.

Reply:
Done :+1: